core/modules/system/css/token.css

```diff
@@ -1,59 +1,170 @@
+
+.tree-grid .token-name {
+  width: 25%;
+}
+
+.tree-grid .token-raw {
   width: 30%;
-  white-space: nowrap;
 }
-.token-tree .token-token {
-  white-space: nowrap;
+
+.tree-grid .token-description {
+  width: 40%;
+}
+
+.tree-grid tr {
+  background: padding-box #aaa;
+  border: 0;
+}
+
+.tree-grid th {
+  border: 0;
+}
+
+.tree-grid td {
+  background: padding-box #fff;
+  border: #aaa solid 0;
+  border-bottom-width: 1px;
+  height: 2.4em;
+  padding: .2em .5em;
+}
+
+.tree-grid .token-name {
+  padding-left: 2.4em;
+}
+
+.tree-grid td:first-child {
+  border-left-width: 24em;
+}
+
+.tree-grid [aria-level="1"] td:first-child {
+  border-left-width: 0;
+}
+
+.tree-grid [aria-level="2"] td:first-child {
+  border-left-width: 2em;
+}
+
+.tree-grid [aria-level="3"] td:first-child {
+  border-left-width: 4em;
+}
+
+.tree-grid [aria-level="4"] td:first-child {
+  border-left-width: 6em;
 }
-.token-tree .token-key {
-  float: left;
+
+.tree-grid [aria-level="5"] td:first-child {
+  border-left-width: 8em;
+}
+
+.tree-grid [aria-level="6"] td:first-child {
+  border-left-width: 10em;
+}
+
+.tree-grid [aria-level="7"] td:first-child {
+  border-left-width: 12em;
+}
+
+.tree-grid [aria-level="8"] td:first-child {
+  border-left-width: 14em;
+}
+
+.tree-grid [aria-level="9"] td:first-child {
+  border-left-width: 16em;
+}
+
+.tree-grid [aria-level="10"] td:first-child {
+  border-left-width: 18em;
+}
+
+.tree-grid [aria-level="11"] td:first-child {
+  border-left-width: 20em;
+}
+
+.tree-grid [aria-level="12"] td:first-child {
+  border-left-width: 22em;
 }
-.token-tree .token-more {
-  float: right;
+
+.tree-grid [aria-level="2"] td {
+  background: #f5f5f5;
 }
-.token-tree .token-description {
-  color: #666;
-  white-space: normal;
-  clear: both;
+
+.tree-grid [aria-level="3"] td,
+.tree-grid [aria-level="4"] td,
+.tree-grid [aria-level="5"] td,
+.tree-grid [aria-level="6"] td,
+.tree-grid [aria-level="7"] td,
+.tree-grid [aria-level="8"] td,
+.tree-grid [aria-level="9"] td,
+.tree-grid [aria-level="10"] td, 
+.tree-grid [aria-level="11"] td,
+.tree-grid [aria-level="12"] td {
+  background: #e9e9e9;
 }
-.token-tree .token-group-description {
+
+.tree-grid button {
+  background: transparent;
+  border: 0;
+  height: 2em;
+  line-height: 0;
+  margin: 0 .2em 0 -2.2em;
+  overflow: hidden;
   padding: 0;
+  position: relative;
+  text-indent: -9999em;
+  white-space: nowrap;
+  width: 2em;
+}
+
+.tree-grid button:after {
+  background: content-box transparent var(--icon-caret-down) center / 50% no-repeat;
+  box-sizing: border-box;
+  content: '';
+  display: block;
+  left: 0;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  transform: rotate(-90deg);
+  transition: .15s;
+  width: 100%;
 }
 
+[aria-expanded="true"] button:after {
+  transform: rotate(0deg);
+}
+
+.tree-grid a:focus {
+  text-decoration: underline;
+}
+
+.token-key.selected-token {
+  font-weight: bold;
+}
+
+.token-key.selected-token:after {
+  content: ' (selected)';
+}
```

Comment:
Can this be placed with JavaScript and `Backdrop.t()`? As this is now it's not translatable.